### ISSUE TITLE
🚀 [Feature] Delete user as admin through API only

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#3038]
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-10T20:21:03Z" title="Monday, February 10th 2020, 9:21:03 pm +01:00">Feb 10, 2020</time>_
_Closed <time datetime="2020-03-10T18:17:01Z" title="Tuesday, March 10th 2020, 7:17:01 pm +01:00">Mar 10, 2020</time>_
---

## :rocket: Feature
This is a required step in order to finish #17. 

### Problem
So, let's say for safety reasons we might not implement #17 at all - but we allow admins to delete users. Admins can use external UIs for a graphQL endpoint e.g. [Postman](https://www.postman.com/).

### Implementation
Basically you only need to change [this line](https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/middleware/permissionsMiddleware.js#L136) into something like this:

```js
      DeleteUser: or(isDeletingOwnAccount, isAdmin),
```
And then provide sufficient backend testing for this critical security feature.

### Design & Layout
None.
